library/solr

```diff
@@ -8,22 +8,27 @@ Maintainers: The Apache Solr Project <dev@solr.apache.org> (@asfbot),
 GitRepo: https://github.com/apache/solr-docker.git
 GitFetch: refs/heads/main
 
-Tags: 9.1.1, 9.1, 9, latest
+Tags: 9.2.0, 9.2, 9, latest
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 40c0ec4e3220faa34d343828b5a7b7aa37fee6f0
+GitCommit: 3b5c1e698660ad1946a538ce820b94a99d5f9408
+Directory: 9.2
+
+Tags: 9.1.1, 9.1
+Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
+GitCommit: 3b5c1e698660ad1946a538ce820b94a99d5f9408
 Directory: 9.1
 
 Tags: 9.0.0, 9.0
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 40c0ec4e3220faa34d343828b5a7b7aa37fee6f0
+GitCommit: 3b5c1e698660ad1946a538ce820b94a99d5f9408
 Directory: 9.0
```

Comment:
Any particular reason to keep these versions listed here? (from what I can tell, only the latest "9.x" is actively supported, and the likelihood of a release like "9.0.1" is very low?  feel free to ignore if that's not the case!)

Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)